### ISSUE TITLE
When spellchecking with IE8, extra paragraphs are created occasionally 

### ISSUE THREAD

**Issue by peter-klaesson-stratsys** [badsyntax/jquery-spellchecker#35]
To reproduce: (use native IE8, not IE9 in IE8 mode)

Go to http://jquery-spellchecker.badsyntax.co/ckeditor.html

Place the cursor between "great explorer" and "of the truth" in the paragraph. Hit enter.

Press spellchecker (ABC-button on the top of the editor).

Spellcheck the second paragraph (starting with 'of the truth'... Correct Noo, consequencses's and painful.

At either "consequencses's" or "painful", the normalize function will create an extra paragraph when normalizing the p-element.

Created a pull request at https://github.com/padolsey/findAndReplaceDOMText/pull/9
that will fix this problem, hopefully.

**Comment by alang** [comment]
The word "painful" is spelled correctly and shouldn't be highlighted to begin with.  Perhaps this is linked to issues in #26?

**Comment by peter-klaesson-stratsys** [comment]
No, #26 is not related to this issue. This issue only affects IE8 and is solved by deferring the text normalization. (It is a work around for some timing issue in IE8)
